### PR TITLE
Devop 94 dockerize app

### PR DESCRIPTION
**Pull request by bettesworthjayna**
# Description

This PR includes the following proposed change(s):

- [DEVOP-94](https://citz-imb.atlassian.net/jira/software/c/projects/DEVOP/boards/18/backlog?view=detail&selectedIssue=DEVOP-94&epics=visible&issueLimit=100) - dockerize application
- Set up TypeScrip and healthCheck routes in express API

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change
- [x] Dependencies added/removed

## Aspect(s) of Project Affected
- [x] Frontend
- [x] API
- [ ] Database
- [ ] Workflows
- [ ] Documentation

## Best Practices Checklist
- [x] I have performed a self-review of my own code
- [x] My code follows the Airbnb React Style Guidelines
- [ ] Documentation has been updated to reflect my changes
- [ ] New and existing tests pass locally with my changes

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- deployed and run docker files
- tested API endpoints at localhost

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+dist
```
Comment (akroon3r) [inline .gitignore:2]:
```suggestion
dist

```

File: src/api/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+RUN npm install
+
+
+# Add typescript
+RUN npm i -D typescript@5.0.4
+
+# Complete JavaScript Build
+RUN npm run build
+
+
```
Comment (akroon3r) [inline src/api/Dockerfile:20]:
```suggestion
# Prod Build
```

File: src/webApp/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# Replace ngnx config files 
+RUN rm /etc/nginx/conf.d/default.conf
+COPY nginx/nginx.conf /etc/nginx/conf.d
+
+# Expose default port
+EXPOSE 80
+
+# Run Nginx server
+CMD ["nginx", "-g", "daemon off;"]
+
```
Comment (akroon3r) [inline src/webApp/Dockerfile:36]:
```suggestion

```

**Review by adamkroonbcgov — COMMENTED** [review]
Looks good minus small suggestions. I'll approve when you've committed them.

**Review by adamkroonbcgov — APPROVED** [review]
🚀

**Pull request merged**